app/controllers/alerts_controller.rb

```diff
@@ -19,8 +22,12 @@ def create
     end
   end
 
+  # Resolves alert if it is resolvable? Requires admin permission.
+  # Normally, same alerts won't be created for the second time.
+  # Resolving alert, allows it to be created for the second time if conditions are met.
   def resolve
-    ensure_alerts_are_enabled { return}
+    ensure_alerts_are_enabled { return }
```

Comment:
This isn't necessary, as this 'alerts are enabled' check is specifically about whether the 'get help button' that allows any user to create a 'NeedHelpAlert' is enabled. Even if that is disabled, other alerts are being created and may be resolved.